src/Context/SoapContext.php

```diff
@@ -102,6 +108,22 @@ public function iRegisterXpathNamespaces(TableNode $namespaces)
     }
 
     /**
+     * @Given /^(?:|I )expect SOAP exception(?:| with code "(\d+)")(?:|( and| or)? with message "([^"]+?)")$/
```

Comment:
Will it make more sense to use @Then keyword for this step? It sounds more like an assertion step then a setup step. I know behat ignores it, but for the user ;)